wallet/lib/utils/extension.dart

```diff
@@ -46,6 +53,13 @@ extension ConvertFromU on String {
     }
     return '';
   }
+  String convertFromUCoin(String amount) {
+    switch (this) {
```

Comment:
why we are using switch here?

Reply:
The 1166 branch already has a function for converting pylon to usd. So I copied the same code from there to avoid any future conflicts.

Reply:
Check my comments.